### DISCUSSION TITLE
@scholzj

### DISCUSSION THREAD

**Discussion by damodarreddy08** [strimzi/strimzi-kafka-operator#9671] [Q&A]
@scholzj 
Ah, that fixed it!
Thanks a lot!

_Originally posted by @j-chao in https://github.com/strimzi/strimzi-kafka-operator/issues/1486#issuecomment-478281584_
 
Kindly share the how to generate the certificates of DNS which is used for kafka domain, means steps are require for certificate generation of secretName: kafka-ssl-secrets-123

**Comment by damodarreddy08** [comment]
hi

**Reply by scholzj** [reply]
You should probably:
* Fix the title
* Stop opening issues for things that clearly are not issues. Why didn't you read and followed the instructions where / how to ask questions?
* Start from the beginning and explain what it is you are asking for. I do not have any guide on how to generate certificates and I'm not sure about what Kafka domain are you talking about.
